public/app/plugins/panel/graph/graph.js

```diff
@@ -352,6 +352,45 @@ function (angular, $, moment, _, kbn, GraphTooltip) {
           };
         }
 
+        //Override min/max to provide more flexible autoscaling
+        function autoscaleSpanOverride(yaxis, data, options) {
+          var m, op, num, precision;
+          if (yaxis.min != null && data != null) {
+            m = yaxis.min.match(/([<=>~]*)\W*(\d+(\.\d+)?)/);
```

Comment:
Should we do:
/([<=>~]?)\W*(\d+(\.\d+)?)/

Instead of:
/([<=>~]*)\W*(\d+(\.\d+)?)/